turbo.json

```diff
@@ -8,16 +8,16 @@
         "^build",
         "generate-model",
         "generate-protobuf",
-        "infra:start"
+        "//#infra:start"
```

Comment:
why this "//#" ?

Reply:
It's a way to tell `turbo` to run a task from the root `package.json`
https://turbo.build/repo/docs/core-concepts/monorepos/running-tasks#running-tasks-from-the-root